src/components/AssetSelection/components/AssetMenuButton.tsx

```diff
@@ -55,7 +50,14 @@ export const AssetMenuButton = ({
       isLoading={isLoading}
       {...buttonProps}
     >
-      <Flex alignItems='center' gap={2} {...flexProps}>
+      <Flex
+        alignItems='center'
+        gap={2}
+        width='100%'
+        overflow='hidden'
+        textOverflow='ellisis'
```

Comment:
```suggestion
        textOverflow='ellipsis'
```

Also you could remove this line as it doesn't seem to change anything

Reply:
Good catch, removed in https://github.com/shapeshift/web/pull/7501/commits/91a5ba96663539984423fdd8341d71f9453ba262